style.css

```diff
@@ -437,3 +455,125 @@ h2.section-title {
   height: 100%;
   z-index: -111;
 }
+
+@media (min-width: 786px) {
+  .container {
+    max-width: 1156px;
+    margin: 0 auto;
+  }
+
+  .nav-links {
+    list-style-type: none;
+    padding: 0;
+    margin: 0;
+    display: flex;
+  }
+
+  .nav-links li {
+    display: flex;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    padding-right: 20px;
+    color: #344563;
+  }
+
+  i.fas.fa-bars {
+    display: none;
+  }
+
+  .work-description {
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+  }
+
+  .headline-wrap {
+    background-image: url(image/back-desktop.svg);
+    background-position: center;
+  }
+
+  .greetings {
+    font-weight: 700;
+    font-size: 48px;
+    max-width: 500px;
+  }
+
+  .headline-profile {
+    max-width: 920px;
+    font-size: 20px;
+  }
+
+  .works-grid-item {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    gap: 20px;
+    align-items: center;
+  }
+
+  .card-content {
+    max-width: 496px;
+  }
+
+  .works-grid-item img {
+    max-width: 544px;
+  }
+
+  .works-grid-item:nth-child(2) img {
+    grid-column: 2/2;
+    grid-row: 1;
+  }
+
+  .works-grid-item:nth-child(2) .card-content {
+    grid-column: 1/2;
+  }
+
+  .works-grid-item:nth-child(4) img {
+    grid-column: 2/2;
+    grid-row: 1;
+  }
+
+  .works-grid-item:nth-child(4) .card-content {
+    grid-column: 1/2;
+  }
+
+  .about-section {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    align-items: center;
+    gap: 10px;
+  }
+
```

Comment:
- [ ] Good job so far. I think it will look good if you add the rounded corner on the right side of the about page. As a software developer, It is **important to pay attention to all details even the small one**s 😉. Take a look at the comparison below.
**Figma**
![image](https://user-images.githubusercontent.com/90222110/186699739-c9294d10-cc9c-4a80-b33e-07a4b1c21b83.png)

**Your design**
![image](https://user-images.githubusercontent.com/90222110/186699873-33336d50-5e4a-4fb1-a386-1176149512e2.png)

**HINT** You may consider using the `border-top-right-radius` on the about section or you may consider using the `clip-path` property on it. Tools like [Clippy](https://bennettfeely.com/clippy/) can do the best job 😉 

- [ ] You may also consider adding more `margin-top` values to the about section so that there is more space between it and the above section. Please refer to the Figma design given 😎

- [ ] And last, please add more ` padding` to your about-section so that texts inside can be pushed somehow in the center like is shown in the Figma design (Refer to the comparison given above). Sticking to the Figma design is among the project requirements as you know 😃 👍